app/src/components/AnalysisPane.js

```diff
@@ -1,126 +1,247 @@
+      <Select
+        mr={2}
+        border={selectedConcept.Score < CONFIDENCE_THRESHOLD ? '2px solid' : '1px solid'}
+        borderColor={borderColor}
+        borderRadius='0'
+        bg='white'
+        value={result.selectedConceptCode}
+        onChange={(e) => onSelectedConceptChange(result.id, e.target.value)}
+        _hover={{ borderColor: borderColor, boxShadow: 'none' }}
+      >
+        {concepts.map(({ Code, Description, Score }) => (
+          <option key={Code} value={Code}>
+            {result.Text} {Code && ' | '} {Code} {Description && ' | '}
+            {Description} &nbsp;&nbsp; {(Score * 100).toPrecision(4)}%
+          </option>
         ))}
-      </dl>
-    </div>
+      </Select>
+
+      {closeIcon}
+    </Flex>
   );
 }
 
-function ResultTable({ results, category, onToggleItem, excludedItems }) {
+function ResultTable({
+  results,
+  category,
+  stage,
+  onToggleItem,
+  excludedItems,
+  onResultDelete,
+  onResultAdd,
+  onSelectedConceptChange,
+}) {
   const filteredResults = useMemo(() => results.filter((r) => r.Category === category), [results, category]);
-  const [open, setOpen] = useState(false);
+  const [inputValue, setInputValue] = useState('');
+  const handleInputChange = (event) => setInputValue(event.target.value);
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    const input = inputValue.trim();
+    if (input !== '') {
+      onResultAdd(input, category);
+      setInputValue('');
+    }
+  };
+
+  const addEntityInputId = `add-${displayNames[category]}`;
+
+  const addIcon = (
+    <IconButton
+      aria-label='Add'
+      type='submit'
+      icon={<AddIcon />}
+      size='xs'
+      isRound
+      isDisabled={stage === STAGE_TRANSCRIBING}
+      border='1px solid #545b64'
+      _hover={{ bg: '#545b64' }}
+      sx={{
+        '&:hover svg': {
+          color: '#fff',
+        },
+      }}
+    />
+  );
 
   return (
-    <div className={s.resultTable}>
-      <h3 className={open ? s.expanded : s.contracted} onClick={() => setOpen((x) => !x)}>
+    <Box mb={4} mx='3em' _first={{ marginTop: '3em' }} _last={{ marginBottom: '3em' }}>
+      <Box
+        as='h1'
+        mb={4}
+        textAlign='left'
+        width='max-content'
+        fontWeight='bold'
+        fontSize='1.2rem'
+        className={highlightClasses[category]}
+      >
         {displayNames[category]}
-      </h3>
-
-      <HeightSlider>
-        {open ? (
-          <div>
-            {filteredResults.map((r) => (
-              <ResultRow result={r} key={r.Id} onToggleItem={onToggleItem} excludedItems={excludedItems} />
-            ))}
-          </div>
-        ) : null}
-      </HeightSlider>
-    </div>
+      </Box>
+
+      <VStack spacing={2}>
+        <FormControl as='form' onSubmit={handleSubmit}>
+          <Flex width='100%' mb={4} alignItems='center'>
+            <VisuallyHidden as='label' htmlFor={addEntityInputId}>
+              Add displayNames[category]
```

Comment:
This should be an expression, not a string

Reply:
Good catch. Fixed it.